src/VisualStudio/CSharp/Impl/ProjectSystemShim/CSharpProjectShim.cs

```diff
@@ -279,15 +293,10 @@ protected override ParseOptions CreateParseOptions(CommandLineArguments commandL
             }
         }
 
-        internal bool CanCreateFileCodeModelThroughProject(string filePath)
-        {
-            return _projectRoot.CanCreateFileCodeModel(filePath);
```

Comment:
Oops: need to restore the call to this. That can live in the TryCreate* method below and just return null.